app/models/contribution.rb

```diff
@@ -35,18 +41,12 @@ def self.valid_fields?(contrib)
       visibility_valid?(contrib)
   end
 
-  # Allowed values for visibility
-  VISIBILITY_VALUES = %w(public private).freeze
-
```

Comment:
So `visibility_valid?` and `status_valid?` did a bunch of data munging for symbol/string compatibility.  Are these methods still required now that the model has validations?

Reply:
Currently, yes, because the `authorship_api` still ties into them.  Conceivably, those uses could be converted once this is in place.

Reply:
Created a follow-up issue for this - https://github.com/sul-dlss/sul_pub/issues/442